gensim/models/atmodel.py

```diff
@@ -0,0 +1,1017 @@
+        (topic_id, topic_probabiity) 2-tuples.
+
+        Ingores topics with probaility less than `eps`.
+
+        Do not call this method directly, instead use `model[author_names]`.
+
+        '''
+        if isinstance(author_names, list):
+            items = []
+            for a in author_names:
+                items.append(self.get_author_topics(a, minimum_probability=eps))
+        else:
+            items = self.get_author_topics(author_names, minimum_probability=eps)
+
+        return items
+
+    def save(self, fname, ignore=['state', 'dispatcher'], *args, **kwargs):
+        """
+        Save the model to file.
+
+        Large internal arrays may be stored into separate files, with `fname` as prefix.
+
+        `separately` can be used to define which arrays should be stored in separate files.
+
+        `ignore` parameter can be used to define which variables should be ignored, i.e. left
+        out from the pickled author-topic model. By default the internal `state` is ignored as it uses
+        its own serialisation not the one provided by `AuthorTopicModel`. The `state` and `dispatcher`
+        will be added to any ignore parameter defined.
+
+
+        Note: do not save as a compressed file if you intend to load the file back with `mmap`.
+
+        Note: If you intend to use models across Python 2/3 versions there are a few things to
+        keep in mind:
+
+          1. The pickled Python dictionaries will not work across Python versions
+          2. The `save` method does not automatically save all NumPy arrays using NumPy, only
+             those ones that exceed `sep_limit` set in `gensim.utils.SaveLoad.save`. The main
+             concern here is the `alpha` array if for instance using `alpha='auto'`.
+
+        Please refer to the wiki recipes section (https://github.com/piskvorky/gensim/wiki/Recipes-&-FAQ#q9-how-do-i-load-a-model-in-python-3-that-was-trained-and-saved-using-python-2)
+        for an example on how to work around these issues.
+        """
+        if self.state is not None:
+            self.state.save(utils.smart_extension(fname, '.state'), *args, **kwargs)
+
+        # make sure 'state' and 'dispatcher' are ignored from the pickled object, even if
+        # someone sets the ignore list themselves
+        if ignore is not None and ignore:
+            if isinstance(ignore, six.string_types):
+                ignore = [ignore]
+            ignore = [e for e in ignore if e] # make sure None and '' are not in the list
+            ignore = list(set(['state', 'dispatcher']) | set(ignore))
+        else:
+            ignore = ['state', 'dispatcher']
+        # TODO: the only difference between this save method and LdaModel's is the use of
+        # "AuthorTopicModel" below. This should be an easy refactor.
+        # Same goes for load method below.
+        super(AuthorTopicModel, self).save(fname, *args, ignore=ignore, **kwargs)
+
+    @classmethod
+    def load(cls, fname, *args, **kwargs):
```

Comment:
Well, almost matches. The difference in the code is `LdaModel` and `AuthorTopicModel` (although I actually forgot to change one line.

Still, the methods in the two classes are very similar. Refactoring should be easy.